### ISSUE TITLE
Plugin not working, makes Homebridge shut down automatically.

### ISSUE THREAD

**Issue by ThyMollusk** [vieira/homebridge-yeelight-wifi#45]
After Homebridge starts, I get the following:

[9/3/2019, 5:21:39 PM] [homebridge-yeelight.yeelight] doing a round of proactive search for known devices.
[9/3/2019, 5:21:39 PM] Homebridge is running on port 44635.
[9/3/2019, 5:21:39 PM] [homebridge-yeelight.yeelight] received advertisement from 002427.
[9/3/2019, 5:21:39 PM] TypeError: Cannot read property 'defaultValue' of null
    at blacklist (/usr/lib/node_modules/homebridge-yeelight-wifi/utils.js:22:11)
    at YeePlatform.buildDevice (/usr/lib/node_modules/homebridge-yeelight-wifi/platform.js:89:20)
    at YeePlatform.handleMessage (/usr/lib/node_modules/homebridge-yeelight-wifi/platform.js:84:10)
    at Socket.emit (events.js:203:13)
    at UDP.onMessage [as onmessage] (dgram.js:853:8)
[9/3/2019, 5:21:39 PM] [homebridge-yeelight.yeelight] received advertisement from 55644d.
[9/3/2019, 5:21:39 PM] TypeError: Cannot read property 'defaultValue' of null
    at blacklist (/usr/lib/node_modules/homebridge-yeelight-wifi/utils.js:22:11)
    at YeePlatform.buildDevice (/usr/lib/node_modules/homebridge-yeelight-wifi/platform.js:89:20)
    at YeePlatform.handleMessage (/usr/lib/node_modules/homebridge-yeelight-wifi/platform.js:84:10)
    at Socket.emit (events.js:203:13)
    at UDP.onMessage [as onmessage] (dgram.js:853:8)
[9/3/2019, 5:21:39 PM] [homebridge-yeelight.yeelight] received advertisement from 55644d.
[9/3/2019, 5:21:39 PM] TypeError: Cannot read property 'defaultValue' of null
    at blacklist (/usr/lib/node_modules/homebridge-yeelight-wifi/utils.js:22:11)
    at YeePlatform.buildDevice (/usr/lib/node_modules/homebridge-yeelight-wifi/platform.js:89:20)
    at YeePlatform.handleMessage (/usr/lib/node_modules/homebridge-yeelight-wifi/platform.js:84:10)
    at Socket.emit (events.js:203:13)
    at UDP.onMessage [as onmessage] (dgram.js:853:8)
[9/3/2019, 5:21:39 PM] [homebridge-yeelight.yeelight] received advertisement from 002427.
[9/3/2019, 5:21:39 PM] TypeError: Cannot read property 'defaultValue' of null
    at blacklist (/usr/lib/node_modules/homebridge-yeelight-wifi/utils.js:22:11)
    at YeePlatform.buildDevice (/usr/lib/node_modules/homebridge-yeelight-wifi/platform.js:89:20)
    at YeePlatform.handleMessage (/usr/lib/node_modules/homebridge-yeelight-wifi/platform.js:84:10)
    at Socket.emit (events.js:203:13)
    at UDP.onMessage [as onmessage] (dgram.js:853:8)
[9/3/2019, 5:21:39 PM] Got SIGTERM, shutting down Homebridge...

Any idea how to fix this?

**Comment by boro55** [comment]
Add this to your config.json

```
"accessories": [],
"platforms": [
        {
            "platform": "yeelight",
            "name": "Yeelight"
        }
]
```

**Comment by RealDyllon** [comment]
Yup, @boro55 's solution worked!

Replace the existing yeelight platforms data with just the `"platform"` and `"name"` fields. 
Start up homebridge, and allow it to bootup. 
when you get  `all known devices found, stopping proactive search.`  without any errors, you can then ctrl+c to kill the process

Then add all the other yeelight fields to your config.json, like `transition`, `defaultvalue`, etc

And now start up homebridge again. it should work fine

**Comment by FlowFrank** [comment]
I've got the same prolbem as @ThyMollusk. I've tried the solution of @boro55, but that doesn't helped me. After I've tried that, homebridge  stopped working totally. I had to delete the configuration and the plugin.

**Comment by RealDyllon** [comment]
Did you remove the default value part of the config? @FlowFrank

**Comment by FlowFrank** [comment]
You mean like that?

{
            "platform": "yeelight",
            "name": "Yeelight"
}

**Comment by FlowFrank** [comment]
This is the result:
`[10/28/2019, 4:45:00 PM] TypeError: Cannot read property 'defaultValue' of null
    at blacklist (/usr/local/lib/node_modules/homebridge-yeelight-wifi/utils.js:22:11)
    at YeePlatform.buildDevice (/usr/local/lib/node_modules/homebridge-yeelight-wifi/platform.js:89:20)
    at YeePlatform.handleMessage (/usr/local/lib/node_modules/homebridge-yeelight-wifi/platform.js:84:10)
    at Socket.emit (events.js:198:13)
    at UDP.onMessage [as onmessage] (dgram.js:628:8)`

**Comment by RealDyllon** [comment]
Can you send the full log? @FlowFrank

**Comment by vieira** [comment]
Sorry about the long delay. I was doing some tests today and ran into the same problem. Should be fixed in 1.1.2.
